CMakeLists.txt

```diff
@@ -10,7 +10,7 @@
 ###############################################################################
 # Minimum CMake version and policies
 ###############################################################################
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.1.0 FATAL_ERROR)
```

Comment:
Indeed, it's important not too break support for 14.04, as we still have many users who report using it (me included).